.bit/responses/1.3-Week 1 Step 3.md

```diff
@@ -43,15 +43,15 @@ Clients communicate with servers with through `HTTP requests`. For example, when
 <summary>❓ What are some examples of request types??</summary>
```

Comment:
Maybe make it to just one question mark at the end so it's less aggressive haha

Reply:
+1